lib/matrix.gd

```diff
@@ -71,13 +71,22 @@ DeclareProperty( "IsGeneralizedCartanMatrix", IsMatrix );
 ##  <Oper Name="IsDiagonalMat" Arg='mat'/>
 ##
 ##  <Description>
-##  returns true if mat has only zero entries off the main diagonal, false
-##  otherwise.
+##  return <K>true</K> if the matrix <A>mat</A> has only zero entries
```

Comment:
```suggestion
##  returns <K>true</K> if the matrix <A>mat</A> has only zero entries
```
We use "returns" not "return" in the vast majority of entries; I always read this as "IsDiagonalMat returns true...". One could also write "Return true" (with an upper case "R"), I guess, if one does not want to indicate that this continues a sentence that started elsewhere. But then the text is addressing the function ("ordering" it to do something), not the reader; which I find a bit odd for documentation (it's fine by me in code comments).